tasks/libs/ciproviders/gitlab_api.py

```diff
@@ -145,6 +145,18 @@ def convert_to_config_node(json_data):
         return json_data
 
 
+def convert_to_dict(config_node):
```

Comment:
You should rename this to `convert_to_std_collections` / something like this as it treats also lists